.github/dependabot.yml

```
@@ -4,22 +1,25 @@
+    directory: "/"
     schedule:
-      interval: 'daily'
-      time: '11:00'
+      interval: "daily"
     commit-message:
-      prefix: 'chore'
-      prefix-development: 'chore'
-      include: 'scope'
-    target-branch: dev
+      prefix: "ci"
+    target-branch: "main"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)